### PR TITLE
Make ffmpeg plugin's default parameters identical to APP

### PR DESCRIPTION
**Pull request by jsunintel**

### CODE REVIEW

**Comment by jsunintel** [comment]
The PR solved https://github.com/OpenVisualCloud/SVT-AV1/issues/172.

**Comment by jbeich** [comment]
> The PR solved #172.

Please, see how to use [autoclose](https://help.github.com/en/articles/closing-issues-using-keywords) feature.

**Comment by jsunintel** [comment]
Updated the commit based on https://github.com/intel/SVT-HEVC/pull/149.

**Comment by jsunintel** [comment]
@hassount @kylophone We have been blocked by this commit not getting merged. Would you please review this and let me know if any issue?

File: ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-av1.patch [hunk 1]
```diff
@@ -524,7 +524,7 @@ index 0000000..7d0e307
 +        { "vbr", NULL, 0, AV_OPT_TYPE_CONST, { .i64 = 1 },  INT_MIN, INT_MAX, VE, "rc" },
 +
 +    { "qp", "QP value for intra frames", OFFSET(qp),
-+      AV_OPT_TYPE_INT, { .i64 = 32 }, 0, 51, VE },
++      AV_OPT_TYPE_INT, { .i64 = 50 }, 0, 51, VE },
```
Comment (tianjunwork) [inline ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-av1.patch]: max qp = 63?

**Comment by jsunintel** [comment]
@tianjunwork @hassount @joelsole @kylophone please kindly review this again.

**Pull request merged**